frontend/views/utils/validators.js

```diff
@@ -1,3 +1,5 @@
 export const nonWhitespace = (value: string): boolean => /^\S+$/.test(value)
 
 export const decimals = (digits: number): ((value: number) => boolean) => (value: number) => Number.isInteger(value * Math.pow(10, digits))
+
+export const noUppercase = (value: string): boolean => /[^A-Z]+/i.test(value)
```

Comment:
This would work if the only characters were English characters, but we need to also test other languages.

I *think* (please double-check me), that something like `value.toLowerCase() === value` might work better. Please check me though on that.

Reply:
@taoeffect `value.toLowerCase() === value` does work with a string that contains non-english characters. update the validator with it.